torch/fx/passes/split_module.py

```diff
@@ -254,11 +265,10 @@ def record_cross_partition_use(def_node : torch.fx.node.Node, use_node : Optiona
         partition.graph.output(output_vals)
 
         # Construct GraphModule for this partition
-        submod_name = f'submod_{partition_name}'
-        base_mod_attrs[submod_name] = torch.fx.graph_module.GraphModule(partition.targets, partition.graph)  # noqa: B950
+        base_mod_attrs[partition.submod_name] = torch.fx.graph_module.GraphModule(partition.targets, partition.graph)  # noqa: B950
```

Comment:
You didn't write this, but I wonder why we're bypassing the line length too long error rather than just formatting the code to conform to the style, lol